client/flutter/assets/content_bundles/protect_yourself.en.yaml

```diff
@@ -14,7 +14,7 @@ contents:
         <b>Cover your mouth and nose</b> with your bent elbow or tissue when you cough or sneeze.
     - image_name: protect-distance
       body_html: |
-        <b>Stay more than</b> 1 meter (3 feet) away from a person who is sick.
+        <b>Stay more than 1 meter</b> (3 feet) away from other people when possible.
     - image_name: protect-mask
       body_html: |
-        Only wear a mask if you or someone you are looking after are ill with COVID-19 symptoms (especially coughing).
+        <b>Wear masks closer than 1 meter</b> (3 feet) in areas where the virus is spreading.
```

Comment:
This is a change in medical content and should be reviewed by WHO @borowiecth .

The grammar doesn't seem to make sense to me.

Reply:
I may be an acceptable rephrasing of:

> Within wider environments where the virus is spreading, masks should be worn by the general public in settings where it is not possible to maintain at least 1 meter from others.
But on the main page there is a less controversial statement:

> Maintain at least a 1-metre distance between yourself and others
I suggest this one instead

But we should remove the sentence "Only wear a mask if you or someone you are looking after...."
And replace it by this statement that is repeated regularly in the press meetings:

> Avoid the 3Cs: spaces that are Closed, Crowded or involve Close Contact.

Reply:
@borowiecth - this section has an image of a mask, so talking about the 3 C's doesn't match up. Perhaps we should have another tile for advice like that. We should also have some specific guidance around masks as that's an important public health measure.

Reply:
@advayDev1 - would this read better to you?

"<b>Wear masks when closer than 1 meter</b> (3 feet) in areas where the virus is spreading."

Reply:
Yeah grammar LGTM with that if WHO approves the content itself.

Reply:
OK for this